src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve2.PlatformNotSupported.cs

```diff
@@ -3129,37 +3129,37 @@ internal Arm64() { }
         /// svint16_t svshllb[_n_s16](svint8_t op1, uint64_t imm2)
         ///   SSHLLB Zresult.H, Zop1.B, #imm2
         /// </summary>
-        public static Vector<short> ShiftLeftLogicalWideningEven(Vector<sbyte> value, [ConstantExpected] byte count) { throw new PlatformNotSupportedException(); }
```

Comment:
Sorry, for `ShiftLeftLogicalWideningEven` and `ShiftLeftLogicalWideningOdd` could you revert them back to count.

Looks like this API ended up in both https://github.com/dotnet/runtime/issues/94015 (with `count`) and https://github.com/dotnet/runtime/issues/94020 (with `shiftAmount`). It shouldn't really have been in the second set.

Given that all the other shift APIs use `count`, then it makes sense we should continue to use `count` for this one too.